### PR TITLE
Fixes blob broadcast working with dead blobbernauts

### LINKED ISSUE #12302: Blob talk power makes dead blobbernauts speak in deadchat
DEAD: blobbernaut gurgles, "fuck you guys"
DEAD: blobbernaut gurgles, "fuck you guys"
DEAD: blobbernaut gurgles, "fuck you guys"
DEAD: blobbernaut gurgles, "fuck you guys"

Hilarious but probably not intended.

### PR DESCRIPTION
**Pull request by JJRcop**
Chewing through "Easy Fix"

Fixes #12302

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1:

File: code/game/gamemodes/blob/powers.dm [hunk 1]
```diff
@@ -217,7 +217,8 @@
 	else
 		usr << "You broadcast with your minions, <B>[speak_text]</B>"
 	for(var/mob/living/simple_animal/hostile/blob_minion in blob_mobs)
-		blob_minion.say(speak_text)
+		if(blob_minion.stat == CONSCIOUS)
```
Comment (Iamgoofball) [inline code/game/gamemodes/blob/powers.dm]: :-1:

**Comment by PKPenguin321** [comment]
fun removal tag when

**Comment by RemieRichards** [comment]
:-1:, it's mine and goof's accidental feature of fun

**Comment by Iamgoofball** [comment]
i'm gonna make a revert PR if some fucker dares merge this

**Comment by phil235** [comment]
:+1:

**Comment by Iamgoofball** [comment]
fuck off phil you just hate fun dont you

**Pull request merged**